tests/platform/windows/windows-jaws-testSpec.js

```diff
@@ -47,19 +68,64 @@ gpii.tests.windows.jaws = [
+                            "Options.PrimarySynthesizer": "eloq",
+                            "enu-Global.Rate": 40,
+                            "enu-Global.Punctuation": 2,
+                            "enu-Global.Pitch": 75,
+                            "enu-Global.Volume": 50,
+                            "enu-Global.SynthLangString": "French",
+                            "enu-Message.Rate": 40,
+                            "enu-Message.Punctuation": 2,
+                            "enu-Message.Pitch": 75,
+                            "enu-Message.Volume": 50,
+                            "enu-Message.SynthLangString": "French",
+                            "enu-Keyboard.Rate": 40,
+                            "enu-Keyboard.Punctuation": 2,
+                            "enu-Keyboard.Pitch": 75,
+                            "enu-Keyboard.Volume": 50,
+                            "enu-Keyboard.SynthLangString": "French",
+                            "enu-PCCursor.Rate": 40,
+                            "enu-PCCursor.Punctuation": 2,
+                            "enu-PCCursor.Pitch": 75,
+                            "enu-PCCursor.Volume": 50,
+                            "enu-PCCursor.SynthLangString": "French",
+                            "enu-JAWSCursor.Rate": 40,
+                            "enu-JAWSCursor.Punctuation": 2,
+                            "enu-JAWSCursor.Pitch": 75,
+                            "enu-JAWSCursor.Volume": 50,
+                            "enu-JAWSCursor.SynthLangString": "French",
+                            "enu-MenuAndDialog.Rate": 40,
+                            "enu-MenuAndDialog.Punctuation": 2,
+                            "enu-MenuAndDialog.Pitch": 75,
+                            "enu-MenuAndDialog.Volume": 50,
+                            "enu-MenuAndDialog.SynthLangString": "French"
+                        },
+                        "options": {
+                            "filename": "${{environment}.APPDATA}\\Freedom Scientific\\JAWS\\17.0\\Settings\\VoiceProfiles\\GPII.VPF"
+                        }
+                    },
+
+                    {
+                        "settings": {
+                            "Voice Profiles.ActiveVoiceProfileName": "GPII",
+                            "Braille.BrailleMode": 1,
+                            "options.SayAllMode": 1,
+                            "options.SayAllIndicateCaps": false,
+                            "options.SayAllIgnoreShiftKeys": false,
+                            "options.TypingEcho": 3
                         },
                         "options": {
-                            "filename": "${{environment}.APPDATA}\\Freedom Scientific\\JAWS\\15.0\\Settings\\VoiceProfiles.ini"
+                            "filename": "${{environment}.APPDATA}\\Freedom Scientific\\JAWS\\17.0\\Settings\\enu\\DEFAULT.JCF"
                         }
                     }
                 ]
```

Comment:
This is awesome! It would be awesome to test some other values for the common terms (ie. to test more cases in your transformations)  - could you perhaps add one more acceptance test (ie. another NP set, and another block with expected results in this file) that contains some different values for the preferences - just to get a bit broader on the testing